src/hyperion/external_interaction/callbacks/xray_centre/ispyb_mapping.py

```diff
@@ -25,7 +28,7 @@ def populate_xz_data_collection_info(detector_params) -> DataCollectionInfo:
     return info
 
 
-def populate_xy_data_collection_info(detector_params):
+def populate_xy_data_collection_info(detector_params: DetectorParams):
```

Comment:
Nit: Could you add a type hint on the return whilst we're here?

Reply:
nah, and in fact I will remove it from the other function to save two lines; pyright can tell that `return DataCollectionInfo(...)` returns a `DataCollectionInfo` and we aren't using mypy here